### DISCUSSION TITLE
Visual Studio não reconhece que é uma URI - Curso Fundamentos de ASP.NET 6

### DISCUSSION THREAD

**Discussion by LuizH1359** [balta-io/forum#414] [Tira Dúvidas] [answered]
Boa noite, estou realizando o curso, porém estou utilizando o visual studio 2022,  e nessa parte da aula em que colocamos {id:int} o visual studio não está reconhecendo como uma uri e sim como um template, então não está associando como um tipo ou id, está colocando como se fosse o nome , e já procurei porém não encontrei como desativar essa função ou mesmo alterar para uri, poderia me ajudar ?

![duvida template](https://github.com/balta-io/forum/assets/94483864/72dac155-2ec1-4aa4-946f-0e157c600ec0)
![template2](https://github.com/balta-io/forum/assets/94483864/23beabb8-e86d-4381-ad34-9560d0a38004)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @LuizH1359, como vai?
Mas nesse caso é um template mesmo! 

Quando temos uma literal, sem interpolação ou parâmetros, ele considera uma URI, quando tem um parâmetro ou interpolação ele considera um template de rota.
